test/tags/computeColumn.js

```diff
@@ -54,24 +54,4 @@ describe("tags/computeColumn", function () {
       expect(ctx.someOtherVariable).to.equal(10);
     });
   });
-
-  it("should throw error if $$answer is not present", function () {
```

Comment:
instead of removing...let's make it `should not throw error if $$answer is not present`?